daemon/graphdriver/devmapper/deviceset.go

```diff
@@ -141,7 +146,9 @@ type Status struct {
 	// UdevSyncSupported is true if sync is supported.
 	UdevSyncSupported bool
 	// DeferredRemoveEnabled is true then the device is not unmounted.
-	DeferredRemoveEnabled bool
+	DeferredRemoveEnabled      bool
```

Comment:
So RemoveEnabled tells whether deferred device deactivation is enabled or not. And DeleteEnabled tells whether deferred deletion is enabled or not. "Remove" and "Delete" are pretty close terms and it is very confusing. In the context of device mapper, they refer to different operations.